spec/features/forms/form_item/questionings_form_spec.rb

```diff
@@ -170,6 +189,7 @@
       visit(edit_qing_path)
       expect_editable("required", true)
       expect_editable("hidden", true)
+      expect_editable("disabled", true)
```

Comment:
A quick-ish way to simplify this file, which is getting a bit unwieldy, might be to combine `expect_visible` and `expect_editable` into one call, `expect_field_visibility("foo", value)`, where `value` is one of `:hidden`, `:read_only`, or `:editable`. Thoughts?